shared_model/backend/protobuf/transaction.hpp

```diff
@@ -0,0 +1,125 @@
+/**
```

Comment:
same. proto_transaction makes it more consistent with other files

Reply:
I agree with @l4l , since `/protobuf/` in path already specifies the type.